examples/radish34/contracts/__tests__/ERC1820Registry.test.js

```diff
@@ -0,0 +1,62 @@
+import { ethers, utils } from 'ethers';
+import { getWallet, getAccounts, getSigner } from '../src/utils';
+import ERC1820RegistryArtifact from '../artifacts/ERC1820Registry.json';
+import SampleERC1820ImplementerArtifact from '../artifacts/SampleERC1820Implementer.json';
```

Comment:
This artifact does not exist for me after running `npm run build`, causing this test suite to crash.

Reply:
removed this redundant file